### ISSUE TITLE
Version 1.0.0 crashes on Python 3.5

### ISSUE THREAD

**Issue by ckleemann** [zostera/django-bootstrap4#168]
Version 1.0.0 is crashing with Django 2.2.4 on Python 3.5 on Debian. Running ```python manage.py check``` fails. Python 3.5 is still supported by Django 2.2 series so it should also be supported by django-bootstrap4.

Stacktrace:
```
$ python manage.py check
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/site-packages/django/template/utils.py", line 66, in __getitem__
    return self._engines[alias]
KeyError: 'django'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "manage.py", line 15, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.5/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.5/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.5/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.5/site-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python3.5/site-packages/django/core/management/commands/check.py", line 64, in handle
    fail_level=getattr(checks, options['fail_level']),
  File "/usr/local/lib/python3.5/site-packages/django/core/management/base.py", line 390, in check
    include_deployment_checks=include_deployment_checks,
  File "/usr/local/lib/python3.5/site-packages/django/core/management/base.py", line 377, in _run_checks
    return checks.run_checks(**kwargs)
  File "/usr/local/lib/python3.5/site-packages/django/core/checks/registry.py", line 72, in run_checks
    new_errors = check(app_configs=app_configs)
  File "/usr/local/lib/python3.5/site-packages/django/contrib/admin/checks.py", line 79, in check_dependencies
    for engine in engines.all():
  File "/usr/local/lib/python3.5/site-packages/django/template/utils.py", line 90, in all
    return [self[alias] for alias in self]
  File "/usr/local/lib/python3.5/site-packages/django/template/utils.py", line 90, in <listcomp>
    return [self[alias] for alias in self]
  File "/usr/local/lib/python3.5/site-packages/django/template/utils.py", line 81, in __getitem__
    engine = engine_cls(params)
  File "/usr/local/lib/python3.5/site-packages/django/template/backends/django.py", line 25, in __init__
    options['libraries'] = self.get_templatetag_libraries(libraries)
  File "/usr/local/lib/python3.5/site-packages/django/template/backends/django.py", line 43, in get_templatetag_libraries
    libraries = get_installed_libraries()
  File "/usr/local/lib/python3.5/site-packages/django/template/backends/django.py", line 108, in get_installed_libraries
    for name in get_package_libraries(pkg):
  File "/usr/local/lib/python3.5/site-packages/django/template/backends/django.py", line 121, in get_package_libraries
    module = import_module(entry[1])
  File "/usr/local/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 985, in _gcd_import
  File "<frozen importlib._bootstrap>", line 968, in _find_and_load
  File "<frozen importlib._bootstrap>", line 957, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 693, in exec_module
  File "<frozen importlib._bootstrap_external>", line 799, in get_code
  File "<frozen importlib._bootstrap_external>", line 759, in source_to_code
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/usr/local/lib/python3.5/site-packages/bootstrap4/templatetags/bootstrap4.py", line 928
    raise ValueError(f"Pagination pages_to_show should be a positive integer, you specified {pages_to_show}.")
                                                                                                            ^
SyntaxError: invalid syntax
```

**Comment by dyve** [comment]
Thanks. Either we'll fix this or we'll exclude 3.5 from supported versions.

Your best bet is to upgrade to 3.6.

A PR that fixes 3.5 support would also be welcome.

**Comment by dyve** [comment]
Fixed.
